tests/testthat/test-methods.R

```diff
@@ -57,8 +64,9 @@ test_that("as.data.frame.lints", {
     type = c("style", "error"),
     message = c("", "Under no circumstances is the use of foobar allowed."),
     line = c("", "a <- 1"),
-    linter = c("", "custom_linter"),
-    stringsAsFactors = FALSE)
+    linter = c(NA_character_, NA_character_), # These are assigned in lint() now.
```

Comment:
shouldn't `as.data.frame.lints` just exclude this column now? Or keeping it for now for backwards compatibility?

Reply:
It's still very useful, as it doesn't contain NAs when the `lints` come from an actual call to `lint()`.